doc/api/child_process.markdown

```diff
@@ -45,8 +45,8 @@ Note that the `exit`-event may or may not fire after an error has occurred. If
 you are listening on both events to fire a function, remember to guard against
 calling your function twice.
 
-See also [`ChildProcess#kill()`](#child_process_child_kill_signal) and
-[`ChildProcess#send()`](#child_process_child_send_message_sendhandle).
+See also [`ChildProcess.kill()`](#child_process_child_kill_signal) and
+[`ChildProcess.send()`](#child_process_child_send_message_sendhandle).
```

Comment:
The method signature changed and this should now be `#child_process_child_send_message_sendhandle_callback`  See e.g. https://nodejs.org/api/child_process.html#child_process_child_send_message_sendhandle_callback